src/registry.jl

```diff
@@ -49,8 +49,8 @@ mutable struct ResourcesRegistry
 end
 
 get_dash_dependencies(registry::ResourcesRegistry, prop_check::Bool) = prop_check ?
-                                                                    registry.dash_dependency[:dev] :
-                                                                    registry.dash_dependency[:prod]
```

Comment:
off the current `master`

![image](https://github.com/plotly/DashBase.jl/assets/6675409/75235968-ab37-4404-a338-bdd8d58e21ff)

than off this branch

![image](https://github.com/plotly/DashBase.jl/assets/6675409/86fd99c0-87a2-41b3-bfe0-3a96c654a841)

Nice!

---

@sjkelly what about the other patched lines above? Did you use JET to figure out which lines to patch, or are the other patched line just code cleanups?